### ISSUE TITLE
[BUG] Battery and power icon 

### ISSUE THREAD

**Issue by claymorwan** [EliverLara/kde_controlcentre#9]
**Describe the bug**
The battery and power menu icons renders in a weird way
![image](https://github.com/user-attachments/assets/b42eec6f-9359-4516-8697-14bf785394a4)

**To Reproduce**
Idk it kinda just happened upon installation of the widget

**Expected behavior**
Shows correctly

**Screenshots**

**Desktop (please complete the following information):**
 - KDE Plasma Version: 6.1.5
 - Distribution: Arch Linux 6.11.1-arch1-1 (64-bits)
 - Qt Version: 6.7.3
 - KDE Frameworks Version: 6.6.0

**Have you installed the following dependencies? (Yes/No)**
 - `kdeplasma-addons`: Yes
 - `plasma-nm`: Yes
 - KDE Connect: Yes

**Additional context**
I'm on desktop so it could kinda explain why the battery one is dying.

**Comment by EliverLara** [comment]
Hi, I'm sorry for the lata response, Does the battery icon work on your system tray?

**Comment by claymorwan** [comment]
> Hi, I'm sorry for the lata response, Does the battery icon work on your system tray?

Nah, I think cuz I'm on desktop, which is probably the reason why it's doing that since it works just fine on my laptop. Would be nice to have an icon replacing the bugged battery one for that case

**Comment by claymorwan** [comment]
> > Hi, I'm sorry for the lata response, Does the battery icon work on your system tray?
> 
> Nah, I think cuz I'm on desktop, which is probably the reason why it's doing that since it works just fine on my laptop. Would be nice to have an icon replacing the bugged battery one for that case

**Comment by EliverLara** [comment]
It was fixed some days ago, you can download the latest version from [kde-store](https://store.kde.org/p/2196105/) :)

**Comment by claymorwan** [comment]
Ye seems to be working yay, now there's just the power and night light icons that freaks out
